nix/deps/nodejs-patched/default.nix

```diff
@@ -106,6 +107,18 @@ stdenv.mkDerivation {
           '[RCTConvert UIColor:options.tintColor() ? @(*options.tintColor()) : nil];'
   '';
 
+# Patch React Native Xcode Script that searches for nvm
+# FIXME: Remove this once we upgrade react-native to 0.69.x
+    patchReactNativeXcodeScriptPhase = ''
+      substituteInPlace ./node_modules/react-native/scripts/find-node.sh --replace \
+              '# Define NVM_DIR and source the nvm.sh setup script' \
+              '<<comment # \n Define NVM_DIR and source the nvm.sh setup script'
```

Comment:
This `\n` newline will not work. Also, it should be before the hash character, not after.

Reply:
This should work better:
```diff
diff --git a/nix/deps/nodejs-patched/default.nix b/nix/deps/nodejs-patched/default.nix
index 3278cd0c21..d46a14acc8 100644
--- a/nix/deps/nodejs-patched/default.nix
+++ b/nix/deps/nodejs-patched/default.nix
@@ -112,11 +112,11 @@ stdenv.mkDerivation {
     patchReactNativeXcodeScriptPhase = ''
       substituteInPlace ./node_modules/react-native/scripts/find-node.sh --replace \
               '# Define NVM_DIR and source the nvm.sh setup script' \
-              '<<comment # \n Define NVM_DIR and source the nvm.sh setup script'
+              '<<MULTI_LINE_COMMENT${"\n"}# Define NVM_DIR and source the nvm.sh setup script'
 
       substituteInPlace ./node_modules/react-native/scripts/find-node.sh --replace \
               '# Set up the nodenv node version manager if present' \
-              'comment # \n Set up the nodenv node version manager if present'
+              'MULTI_LINE_COMMENT${"\n"}# Set up the nodenv node version manager if present'
     '';
```